osbuild-composer.spec

```diff
@@ -326,7 +326,7 @@ The core osbuild-composer binary. This is suitable both for spawning in containe
 Summary:    The worker for osbuild-composer
 Requires:   systemd
 Requires:   qemu-img
-Requires:   osbuild >= 37
+Requires:   osbuild >= 41
 Requires:   osbuild-ostree >= 37
```

Comment:
I think I would prefer these two to stay in sync. Maybe let's take this opportunity and factor them out?